ui/k-toolbar/main.kytos

```diff
@@ -1,20 +1,92 @@
 <template>
   <k-toolbar-item icon="compass" tooltip="Napp Pathfinder">
-    <k-accordion>
-      <k-accordion-item title="Best Path">
-        <k-dropdown icon="circle-o" title="Source:" :options="get_interfaces"
+    <div style="overflow-y: auto; height:550px;">
```

Comment:
The height property can be more responsive using:
height: calc(100vh - 60px);
It is not the best solution, but it will solve the blank lines below the menu, and the height will be dynamic

Reply:
I added your changes in the latest push. Thank you!